DashKit.xcodeproj/project.pbxproj

```diff
@@ -912,6 +953,24 @@
 			};
 			name = Release;
 		};
+		C11615F1239588DB0039A072 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = UY678SP37Q;
```

Comment:
I believe the `DEVELOPMENT_TEAM` should be `""`.

Reply:
I don't think we need to have special rules for this, as we do with repos intended to be shared with the DIY community. Would prefer to keep it this way to streamline running tests.

Reply:
And yes, this branch is the dash-sdk-update branch; added the Cartfile target as a tagalong.